src/language/dafnyInstallation.ts

```diff
@@ -57,8 +57,13 @@ export function getLanguageServerRuntimePath(context: ExtensionContext): string
   return path.join(context.extensionPath, configuredPath);
 }
 
-function getConfiguredLanguageServerRuntimePath(): string | null {
-  return Configuration.get<string | null>(ConfigurationConstants.LanguageServer.RuntimePath);
+function getConfiguredLanguageServerRuntimePath(): string {
+  const languageServerOverride = process.env['DAFNY_LANGUAGE_SERVER'] ?? '';
+  if(languageServerOverride) {
+    window.showInformationMessage('Using Dafny language server configured in $DAFNY_LANGUAGE_SERVER');
```

Comment:
Since this is targeted to developers, I think we can 1) make the text shorter, and 2) use the space for giving the value of DAFNy_LANGUAGE_SERVER. I have two dafny folder on my machine, dafny and dafny2, so I would feel good getting a confirmation that it picked up the correct dafny folder.

```suggestion
    window.showInformationMessage(`Using $DAFNY_LANGUAGE_SERVER = ${languageServerOverride} for the server path`);
```